src/mmw/apps/modeling/mapshed/calcs.py

```diff
@@ -203,6 +205,38 @@ def manure_spread(aeu):
 
     return n_list, p_list
 
+def ls_factors(lu_strms, areas, avg_slope):
+    results = [0.0] * len(lu_strms)
+    for i in range(len(lu_strms)):
+        results[i] = ls_factor(lu_strms[i] * KM_PER_M, areas[i], avg_slope)
+    return results
+
+def ls_factor(stream_length, area, avg_slope):
+    # units --> stream_length:km, area:hectare, avg_slope:%
+    slope_cell_size = 30 #for posterity
+    if 0 <= avg_slope <= 1.0:
+        m = 0.2
```

Comment:
In that case, would the values have to be passed into the method? That would mildly hurt clarity, but probably not much since ls_factors is only a couple lines away. The expense in cycles is pretty negligible I would imagine, so I guess it's just personal preference where the declarations are, and I don't have a strong opinion either way.